plugins/docs-tools/skills/code-evidence/SKILL.md

```
@@ -0,0 +1,108 @@
+  --query "<QUERY>" \
+  --limit <LIMIT> \
+  --filter-paths <FILTER_PATHS>
+```
+
+If `--reindex` was provided, add `--reindex` to the command.
+
+### 3. Present results
+
+Parse the JSON output and present results to the user in a readable format:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)